### PR TITLE
Add CheckDid's codes.

### PR DESCRIPTION
**Pull request by zhaowanli2**
Add CheckDid's codes.

### CODE REVIEW

File: Sources/Elastos/Blockchain/CheckDid/Elastos.CheckDID.car [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//=========================================================================
+
+module
+{
+    importlib("Elastos.CoreLibrary");
+
+    interface ICheckDID {
```
Comment (songsjun) [inline Sources/Elastos/Blockchain/CheckDid/Elastos.CheckDID.car]: ICheckDID ==> IDIDInspector
CCheckDID ==> CDIDInspector

PublicKeyToIdAddress ==> CheckDID([in] publicKey, [in] did, [out] matched) 
VerifyByPublicKey ==> CheckSign(publicKey, message, signature, [out] matched)

File: Sources/Elastos/Blockchain/CheckDid/Key.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		bool Key::verifyByPublicKey(const std::string &publicKey, const UInt256 &messageDigest,
+									const CMBlock &signature) {
+			CMemBlock<char> mbcPubKey;
+			mbcPubKey.SetMemFixed(publicKey.c_str(), publicKey.size() + 1);
+			CMBlock pubKey = Str2Hex(mbcPubKey);
+
+			return BTCKey::ECDSA65Verify_sha256(pubKey, messageDigest, signature, NID_X9_62_prime256v1);
+		}
+
+		CMBlock Key::getPubKeyFromPrivKey(const CMBlock &privKey, int nid) {
```
Comment (songsjun) [inline Sources/Elastos/Blockchain/CheckDid/Key.cpp]: Are you sure you need this function?
getPubKeyFromPrivKey is accessing the private key and should not be needed

Comment (zhaowanli2) [reply Sources/Elastos/Blockchain/CheckDid/Key.cpp]: OK

File: Sources/Elastos/Blockchain/CheckDid/Elastos.CheckDID.car (hunk 1 above)
Comment (zhaowanli2) [reply Sources/Elastos/Blockchain/CheckDid/Elastos.CheckDID.car]: OK

**Pull request closed without merging**